frontend/pendingTranslations.ftl

```diff
@@ -1,7 +1,21 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-   
+
+# Mobile menu icons 
+
+menu-upgrade-button = Upgrade 
+menu-toggle-open = Open menu
+menu-toggle-close = Close menu
+nav-dashboard = Dashboard
+nav-settings = Settings
+nav-support = Help and Support 
+nav-sign-out = Sign Out
+nav-contact = Contact Us
+
+nav-menu-mobile = Mobile Menu
```

Comment:
Typically the user will already know they're on mobile - or even worse, they may be on desktop but just made the window smaller - so no need to include "Mobile" in the description. But as mentioned where it's used, the label's not needed in general.

```suggestion
```